### ISSUE TITLE
Assignment 12

### ISSUE THREAD

**Issue by jisaacks** [tiy-tpa-js-q1-2015/assignments#115]
https://github.com/tiy-tpa-js-q1-2015/assignments/tree/master/012#021715

**Comment by mewilkerson** [comment]
1) Blog Link: https://medium.com/@mwilk2CB/stress-test-5b95bedf6dc1
2) I'm turning in the farthest I got with assignment 4. It still needs a lot of work, but here is where I left it: https://github.com/mewilkerson/tiy-assignment-4/tree/gh-pages
3) I made the changes you recommended to the JS-week-1-proj (on both the excercises and answers files): https://github.com/mewilkerson/JS-week-1-proj/tree/gh-pages
4) I've been working on the JS-Day-2 work (Assignment 6 I think). I got problem #7 to work correctly, but I still haven't figured out how to make the rot 13 problem work. When we start adding in layer on layer on layer of code, I am having a hard time following the logic and reading the code. I guess the point is to keep at it until I'm fluent in it. Def not there yet. https://github.com/mewilkerson/JS-Day-2
